python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -75,6 +75,9 @@ class AutoMaterializeAssetEvaluation(NamedTuple):
         Sequence[AutoMaterializeCondition],
         Sequence[Tuple[AutoMaterializeCondition, PartitionsSubset]],
     ]
+    num_requested: int
```

Comment:
ok last naming thing (I promise), but because these things will be annoying to rename once we start writing them to the db, I'm wondering if we should rename `AutoMaterializeDecisionType.MATERIALIZE` to `AutoMaterializeDecisionType.REQUEST`.

This aligns with the naming here, and also indicates what we actually plan to do. If the request fails to materialize the asset (failure or otherwise), then we continue on, we don't continue trying to materialize it.

Minor thing, as this ultimately isn't user facing, so feel free to discard

Reply:
I think AutoMaterializeDecisionType.MATERIALIZE is clearer than AutoMaterializeDecisionType.REQUEST. `num_requested` is just shorthand for `num_materializations_requested`

Reply:
I buy it